recipe/meta.yaml

```diff
@@ -1,65 +1,86 @@
 {% set version = "1.5.5" %}
-{% set sha256 = "1fdfd5019a0b423259d91d77c9c8969bc3c2be59a3a13731904331a3257ac05f" %}
+{% set sha256 = "20d9451f59a608e70c93a21c22bc3081f9a1982d90ab50c2effb17fc5cbde577" %}
 
 package:
   name: cyclus
   version: {{ version }}
 
 source:
-  fn: cyclus-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/cyclus/cyclus/archive/{{ version }}.tar.gz
+  fn: cyclus-{{ version }}-{{ sha256 }}.zip
+  url: https://github.com/bennibbelink/cyclus/archive/refs/heads/pyinstall-cmake.zip
```

Comment:
There is a version of this that uses a repo & branch/sha.  Should we switch to that? Does it make a difference?

Reply:
Yes, we should switch to that when we have a tag to reference in the main cyclus repo.  This points to a branch on my fork that has some CMake changes - there was an issue installing the python package from `cyclus/main` in this feedstock.  My understanding is as follows:
- Conda has environment variables to define the installation location (e.g. PREFIX, SP_DIR) to support the portability of our package. These variables are kept as placeholders throughout the build process (you can see this in the install location in the build output - `Installing <file> to PREFIX/bin`)
- In the cyclus build we currently generate `SetupPyInstall.cmake` based on some vars in the build.  I think because we configure this during the build (substitute the @@ variables) the placeholder used by conda was not being kept as a placeholder, rather it was expanded out to the absolute path instead of `SP_DIR/cyclus` (as I saw in build output)

My solution in `bennibbelink:pyinstall-cmake` eliminates the need for configuring a separate CMake script.  I will make a PR in cyclus so you can review there.

Reply:
https://github.com/cyclus/cyclus/pull/1737 has been merged - this now references the `main` branch and is passing.  After release we can change it to reference the release tag.